pypeit/scripts/identify.py

```diff
@@ -54,30 +54,23 @@ def main(args):
         from pypeit.images.buildimage import ArcImage
 
         # Load the MasterArc file
-        if os.path.exists(args.arc_file):
-            arcfil = args.arc_file
-        else:
+        try:
```

Comment:
If the issue here is the filename and/or whether the file exists, I'm not convinced try: except: is the proper way to code this. If you know there could be ambiguity in the filename then I think it is better coding practice to explicitly deal with that. The try except is not really necessary at all here anywhere if ArcImage.from_file delivers sensible error messages.

Reply:
Okay, I now reduced this to `msarc = ArcImage.from_file(args.arc_file)`.  The script doesn't allow the same flexibility in the slit-trace master file, so I'm removing the flexibility to do so here.  `DataContainer` does check that the files exist before it tries to open them and will throw a sensible error.